Sources/Sentry/SentryLogOutput.m

```diff
@@ -6,7 +6,14 @@ @implementation SentryLogOutput
 
 - (void)log:(NSString *)message
 {
+#if defined(TEST) || defined(TESTCI)
+    static NSISO8601DateFormatter *df;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{ df = [[NSISO8601DateFormatter alloc] init]; });
+    printf("%s: %s\n", [df stringFromDate:[NSDate date]].UTF8String, message.UTF8String);
+#else
```

Comment:
`h`: I would suggest adding a protocol for the `SentryLogOutput`, which we can do because it is not public. We can rename this class to `SentryDefaultLogOutput` and let it implement the `SentryLogOutput` protocol. In the tests, we can swap the `SentryDefaultLogOutput` with `SentryBreadcrumbTestLogOutput`, which contains this logic. With that, we can always add new log outputs without changing the existing ones. What do you think, @armcknight?

Reply:
Sounds like a nice idea, I'll look into it.

Reply:
I'm not sure where this would be set in the tests. Seems like it would need to be done for every test class? I think we should leave it like it is for now, and if we find a need for the setup you describe later, let's do it then.

Reply:
Ideally, we would have a base test class or something. For now, we could just use the `+ (void)load` method for some global test setup and change the log output there. What do you think?